### DISCUSSION TITLE
posts/upgrade-proxmox-to-8/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#260] [Comments]
# posts/upgrade-proxmox-to-8/

Proxmox 8.0 has been releases (June, 22, 2023) and includes several new features and a Debian version upgrade. Among the changes are:

https://technotim.live/posts/upgrade-proxmox-to-8/

**Comment by RKolibri** [comment]
## Small typo error
Proxmox 8.0 has been `releases` (June, 22, 2023) and includes several new features and a Debian version upgrade. Among the changes are:

`releases` should be `released`

**Reply by timothystewart6** [reply]
Fixed!  Thank you!

**Comment by bigsk1** [comment]
I just googled new cpu on proxmox 8 and bingo my man techno tim at #1 spot, haha nice. Not sure if you did a video yet on this but just followed the guide on proxmox site and was able to upgrade without issues, always a little worried when doing these kind of upgrades. Keep up the great work.

**Reply by timothystewart6** [reply]
Nice!  Thank you!!!

**Comment by Zareix** [comment]
Hi, in « Upgrade APT Packages », the command is `apt dist-upgrade` but I think you meant to write `apt upgrade`

**Reply by timothystewart6** [reply]
Hi!  I think we actually want to do a `dist-upgrade` here to be sure all packages are upgraded before we upgrade to 8.

**Reply by Zareix** [reply]
Oh I misunderstood the command chronology 😅
Well perfect. And thanks for the tutorial, didn't even know PVE8 was out at first

**Comment by yuriw** [comment]
Do you have steps for the backup server ad well Upgrade from 2 to 3?

Thx for this, I will try to follow these steps soon.

**Comment by yuriw** [comment]
Well after details preparation I messed up my upgrade :(
How ?  
See https://forum.proxmox.com/threads/v8-upgrade-failure-you-are-attempting-to-remove-the-meta-package-proxmox-ve.129407/

Any help how to recover is greatly appreciated.

TIA

**Reply by yuriw** [reply]
bad link sorry https://forum.proxmox.com/threads/failed-upgrade-to-pve-8-need-help.130314/

**Reply by yuriw** [reply]
Fix it!

**Comment by ROC-WDS** [comment]
I was consulting the Oracle about upgrading from PVE 7 to 8 in a cluster and I'm not surprised that Techno Tim had an upgrade guide already fleshed out.  This was as easy to follow as your 11 Things to do After Installing Proxmox :-)

## Another Small Typo

In the [Upgrade the system to Debian Bookworm and Proxmox VE 8.0](https://technotim.live/posts/upgrade-proxmox-to-8/#upgrade-the-system-to-debian-bookworm-and-proxmox-ve-80) section, fourth sentence: 
**This step and take some time depending...** should be **This step may take some time depending...**

**Reply by timothystewart6** [reply]
Thank you!
